Monal/Classes/SwiftuiHelpers.swift

```diff
@@ -681,7 +692,7 @@ class SwiftuiInterface : NSObject {
             case "GeneralSettings" :
                 host.rootView = AnyView(UIKitWorkaround(GeneralSettings()))
             case "ActiveChatsPrivacySettings":
-                host.rootView = AnyView(AddTopLevelNavigation(withDelegate: delegate, to: PrivacySettings()))
+            host.rootView = AnyView(AddTopLevelNavigation(withDelegate: delegate, to: PrivacySettings()))
```

Comment:
Don't randomly remove spaces!

```suggestion
                host.rootView = AnyView(AddTopLevelNavigation(withDelegate: delegate, to: PrivacySettings()))
```